canonical-web-and-design/react-components
# Release 0.17.0

## Done

- Prepare storybook for 0.17.0
- Bump version

## QA

### QA steps

- Check if storybook runs without errors
  - verify [against release notes](https://github.com/canonical-web-and-design/react-components/releases/tag/untagged-ff38565a08c7e7c0bc78) if new things are there
 


### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```
